cmd/neofs-adm/docs/subnetwork-creation.md

```diff
@@ -1,13 +1,13 @@
 # NeoFS subnetwork creation
 
 This is a short guide on how to create NeoFS subnetworks. This guide 
-considers that side chain and inner ring (alphabet nodes) have already 
-been deployed and side chain contains deployed `subnet` contract.
+considers that the sidechain and the inner ring (alphabet nodes) have been already 
```

Comment:
Why, though? I though `have already ...` is more common. Though I have nor arguments except for this random link https://textranch.com/176789/have-been-already/or/have-already-been/

Reply:
For some reason, I've always tended to use it like this. Now, I've explored the issue and probably `have already been...` is better